org.kde.CrowTranslate.yaml

```diff
@@ -0,0 +1,61 @@
+id: org.kde.CrowTranslate
+sdk: org.kde.Sdk
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  # The app supports capturing screen via KWin and Mutter DBus APIs, but
+  # access to it seems overly broad in spite of the app also supporting XDP API
+  #- --talk-name=org.kde.KWin
+  #- --talk-name=org.gnome.Shell
+rename-icon: crow-translate
```

Comment:
```suggestion
  #- --talk-name=org.gnome.Shell
rename-desktop-file: io.crow_translate.CrowTranslate.desktop
rename-icon: crow-translate
```

Reply:
But why? Isn't appid is `org.kde.CrowTranslate`?

Reply:
the desktop file you are installing still has the old name...

Did you build this locally or just checked the 4th checkbox?

Reply:
> the desktop file you are installing still has the old name...

Ah, it's because I using the old commit that corresponds to the version with the old appid. What will be the best way to proceed? Add `rename-desktop-file` for now and remove after the release? Or should I pin to the latest commit?

> Did you build this locally or just checked the 4th checkbox?

I built it on KDE CI, but didn't check after making all these changes, sorry.

Reply:
> Add rename-desktop-file for now and remove after the release? 

Yea that's fine. We can proceed with the build and test. You can remove it later.